test/migrators/module/shadowed_variables.hrx

```diff
@@ -0,0 +1,42 @@
+<==> arguments
+--migrate-deps
+
+<==> input/entrypoint.scss
+@import "library";
+$a : red;
+a {
+  color: $a;
+  $b: red;
+  background: $b;
+}
+
+@function fn($c) {
+  @return $c;
+}
+
+@mixin mixin($d) {
+  color: $d;
+}
+
+<==> input/_library.scss
+$a: blue;
+$b: blue;
+$c: blue;
+$d: blue;
+
+<==> output/entrypoint.scss
+@use "library";
+$a : red;
```

Comment:
Why does this shadow the library's definition rather than assigning to it?

Reply:
Is `$library.a: red;` valid?

Reply:
Ping

Reply:
Oh, sorry! Yes, it is.